### PR TITLE
G0 6292 header pull home

### PR DESCRIPTION
**Pull request by shalinee-walkover**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: apps/web-giddh/src/app/shared/header/header.component.ts [hunk 1]
```diff
@@ -81,6 +81,8 @@ export class HeaderComponent implements OnInit, AfterViewInit, OnDestroy, AfterV
     public imgPath: string = '';
     public subscribedPlan: SubscriptionsUser;
     public isLedgerAccSelected: boolean = false;
+    public asideHelpSupportMenuState: string = 'out';
```
Comment (rohit-walkover) [inline apps/web-giddh/src/app/shared/header/header.component.ts:84]: Doc

File: apps/web-giddh/src/app/shared/header/header.component.ts [hunk 2]
```diff
@@ -81,6 +81,8 @@ export class HeaderComponent implements OnInit, AfterViewInit, OnDestroy, AfterV
...
+    public asideSettingMenuState: string = 'out';
```
Comment (rohit-walkover) [inline apps/web-giddh/src/app/shared/header/header.component.ts:85]: Add doc

File: apps/web-giddh/src/app/shared/header/header.component.ts [hunk 3]
```diff
@@ -253,6 +255,7 @@ export class HeaderComponent implements OnInit, AfterViewInit, OnDestroy, AfterV
     public dateFieldPosition: any = { x: 0, y: 0 };
     /* This will check if company is allowed to beta test new modules */
     public isAllowedForBetaTesting: boolean = false;
+    public isMobileSidebar: boolean = false;
```
Comment (rohit-walkover) [inline apps/web-giddh/src/app/shared/header/header.component.ts:258]: JS doc

File: apps/web-giddh/src/app/shared/header/header.component.ts [hunk 4]
```diff
@@ -775,6 +787,29 @@ export class HeaderComponent implements OnInit, AfterViewInit, OnDestroy, AfterV
         this.showManageGroupsModal();
     }
 
+    public toggleBodyClass() {
```
Comment (rohit-walkover) [inline apps/web-giddh/src/app/shared/header/header.component.ts:790]: return type  and doc

File: apps/web-giddh/src/app/shared/header/header.component.ts [hunk 5]
```diff
@@ -775,6 +787,29 @@ export class HeaderComponent implements OnInit, AfterViewInit, OnDestroy, AfterV
...
+        if (this.asideHelpSupportMenuState === 'in') {
+            document.querySelector('body').classList.add('fixed');
+        } 
+        else {
+            document.querySelector('body').classList.remove('fixed');
+        }
+    }
+
+    public toggleHelpSupportPane(show: boolean): void {
```
Comment (rohit-walkover) [inline apps/web-giddh/src/app/shared/header/header.component.ts:799]: JS doc

File: apps/web-giddh/src/app/shared/header/header.component.ts [hunk 6]
```diff
@@ -1657,4 +1724,8 @@ export class HeaderComponent implements OnInit, AfterViewInit, OnDestroy, AfterV
             this.store.dispatch(this.companyActions.SetApplicationDate(dates));
         }
     }
+
+    public redirectToMobileHome(): void {
```
Comment (rohit-walkover) [inline apps/web-giddh/src/app/shared/header/header.component.ts:1728]: JS doc

File: apps/web-giddh/src/assets/css/style-2.scss [hunk 7]
```diff
@@ -1593,10 +1603,20 @@ modal-container.modal {
...
+    .modal-body{
+        max-width: 1120px !important;
+        width: 100% !important;
+        .calendly-inline-widget object{
+            width: 100% !important;
+            height: 610px !important;
+        }
+    }
 }
+// .calendly-inline-widget object {
```
Comment (rohit-walkover) [inline apps/web-giddh/src/assets/css/style-2.scss:1616]: remove commented code if not required

File: apps/web-giddh/src/app/shared/header/header.component.html [hunk 8]
```diff
@@ -448,29 +460,20 @@ <h6 class="mrB1">MENU
...
     </div>
-
     <div class="mobile_menu_icon" (click)="sideBarStateChange(sideMenu.isopen = !sideMenu.isopen)">
         <span class="icon-bar" [tooltip]="sideMenu.isopen ? 'Collapse' : 'Expand'" placement="right"></span>
     </div>
-
 </div>
 
-<header class="clearfix header-container desktop-header " [ngClass]="{'unvisible': companyList.length===0 , 'menu_open':sideMenu.isopen , 'ledger-page': isLedgerAccSelected }">
+<header class="clearfix header-container desktopHeader" [ngClass]="{'unvisible': companyList.length===0 , 'menu_open':sideMenu.isopen }">
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/shared/header/header.component.html:471]: Avoid camelCase in class names

File: apps/web-giddh/src/app/shared/header/header.component.html [hunk 9]
```diff
@@ -561,36 +563,57 @@ <h4 class="inline text-capitalize" *ngIf="!navigationEnd && !navigateToUser">
...
-<!--                        <li [hidden]="isTodaysDateSelected">-->
-<!--                            <button class="btn btn-link" (click)="jumpToToday();">Today</button>-->
-<!--                        </li>-->
-                        <li class="dateRangPicker sss notToday"  (click)="showGiddhDatepicker($event)" >
+                <div class="lower-layer-header">
+                    <ul class="nav universal-date  pull-left" *ngIf="(selectedCompany | async)">
+                        <!-- <li [hidden]="isTodaysDateSelected" class="border-none pl-0"> 
+                            <button class="btn btn-link" (click)="jumpToToday();">Today</button>
+                        </li> -->
+                        <li class="dateRang-picker not-today"  (click)="showGiddhDatepicker($event)" >
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/shared/header/header.component.html:574]: Correct spelling of dateRang-picker to date-range-picker

File: apps/web-giddh/src/app/shared/header/header.component.scss [hunk 10]
```diff
@@ -447,43 +454,36 @@ a[vr-item]:focus {
...
-    position: absolute;
-    left: 0px;
-    background-color: $governor-bay;
-    padding: 8px 6px 8px 7px;
-    top: 0px;
-    text-align: center;
-    display: inline-flex;
-    color: #fff;
+    padding-right: 15px;
+    border: 1px solid #fff;
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/shared/header/header.component.scss:479]: Avoid hex code in color

File: apps/web-giddh/src/app/shared/header/header.component.scss [hunk 11]
```diff
@@ -638,8 +644,60 @@ ul.sub-list {
...
+    color: $deep-sky-blue !important;
+    font-family: 'opensans-semibold';
+}
+/*notification css*/
+.white-row {
+    background-color: $white;
+    padding: 10px 15px;
+    margin-bottom:10px;
+    font-size: 16px;
+    box-shadow: 1px 1px 3px #00000029;
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/shared/header/header.component.scss:672]: Avoid hex code in color

File: apps/web-giddh/src/app/shared/header/header.component.scss [hunk 12]
```diff
@@ -687,20 +738,28 @@ ul.sub-list {
...
         }
     }
-    .primary-header .btn-group.cp.mrL1.d-inline-block {
-        height: 34px;
-    }
+    
 }
-
 @media (max-width: 767px) {
+.desktopHeader{
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/shared/header/header.component.scss:744]: Avoid camelCase in class names

File: apps/web-giddh/src/app/shared/header/header.component.ts (hunk 1 above)
Comment (prateek-giddh) [inline apps/web-giddh/src/app/shared/header/header.component.ts:84]: Missing JS comments

File: apps/web-giddh/src/app/shared/header/header.component.ts (hunk 4 above)
Comment (prateek-giddh) [inline apps/web-giddh/src/app/shared/header/header.component.ts:790]: Missing JS comments

File: apps/web-giddh/src/app/shared/header/header.component.ts (hunk 5 above)
Comment (prateek-giddh) [inline apps/web-giddh/src/app/shared/header/header.component.ts:799]: Missing JS comments

File: apps/web-giddh/src/app/shared/header/header.component.ts [hunk 13]
```diff
@@ -775,6 +787,29 @@ export class HeaderComponent implements OnInit, AfterViewInit, OnDestroy, AfterV
...
+        this.asideHelpSupportMenuState = show ? 'in' : 'out';
+        this.toggleBodyClass();
+    }
+
+    public toggleSidebarPane(show: boolean, isMobileSidebar: boolean): void {
+        this.isMobileSidebar = isMobileSidebar;
+        this.asideHelpSupportMenuState = 'out';
+        this.asideSettingMenuState = show ? 'in' : 'out';
+        this.toggleBodyClass();
+        document.querySelector('body').classList.toggle('mobile-setting-sidebar');
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/shared/header/header.component.ts:810]: Missing JS comments

File: apps/web-giddh/src/assets/css/style-2.scss [hunk 14]
```diff
@@ -9084,9 +9103,53 @@ ngx-daterangepicker-material {
         }
     }
 }
+/*notofication css*/
+.HW_badge.HW_visible{
```
Comment (prateek-giddh) [inline apps/web-giddh/src/assets/css/style-2.scss:9107]: Avoid using improper names in class names (Applicable to all class names that begin with HW)

File: apps/web-giddh/src/assets/fonts/icomoon/fonts/bkcup-3-jun/icomoon.svg [hunk 15]
```diff
@@ -0,0 +1,110 @@
+<?xml version="1.0" standalone="no"?>
```
Comment (prateek-giddh) [inline apps/web-giddh/src/assets/fonts/icomoon/fonts/bkcup-3-jun/icomoon.svg:1]: Can we not keep this backup on server?

File: apps/web-giddh/src/assets/css/style-2.scss (hunk 14 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/assets/css/style-2.scss:9107]: This is third party classes , we can't change it and can't add new classes in this

File: apps/web-giddh/src/assets/fonts/icomoon/fonts/bkcup-3-jun/icomoon.svg (hunk 15 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/assets/fonts/icomoon/fonts/bkcup-3-jun/icomoon.svg:1]: we can, but every new icon we need to make backup and replace whole file of icomoon

Comment (shalinee-walkover) [reply apps/web-giddh/src/assets/fonts/icomoon/fonts/bkcup-3-jun/icomoon.svg:1]: we need to discuss about it, please ignore for this PR iif you can. will do needful for this

File: apps/web-giddh/src/app/settings/financial-year/financial-year.component.scss [hunk 16]
```diff
@@ -1,3 +1,7 @@
+.section_head{
```
[the remark this answers is not in the document]
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/settings/financial-year/financial-year.component.scss]: done

File: apps/web-giddh/src/app/shared/header/components/aside-setting/aside-setting.component.scss [hunk 17]
```diff
@@ -0,0 +1,262 @@
...
+        transform: rotate(45deg);
+    }
+   
+}
+:host.setting-sidebar{
+    position: fixed;
+    top: 61px;
+    bottom: 0;
+    min-height: 100vh;
+    background: #fff;
```
[the remark this answers is not in the document]
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/shared/header/components/aside-setting/aside-setting.component.scss]: done

File: apps/web-giddh/src/app/shared/header/header.component.html (hunk 8 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/shared/header/header.component.html]: done

File: apps/web-giddh/src/app/shared/header/header.component.scss (hunk 10 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/shared/header/header.component.scss]: done

File: apps/web-giddh/src/app/shared/header/header.component.scss (hunk 11 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/shared/header/header.component.scss:672]: appearing many files

File: apps/web-giddh/src/app/shared/header/header.component.scss (hunk 12 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/shared/header/header.component.scss]: done

File: apps/web-giddh/src/app/mobile-home/mobile-home.component.ts [hunk 18]
```diff
@@ -0,0 +1,389 @@
...
+        this.isLoading = true;
+
+        if (this.listOfSelectedGroups && this.listOfSelectedGroups.length > 0) {
+            let lastGroup = this.generalService.getLastElement(this.listOfSelectedGroups);
+            this.commandKRequestParams.group = lastGroup.uniqueName;
+        } else {
+            this.commandKRequestParams.group = "";
+        }
+
+        this.commandKService.searchCommandK(this.commandKRequestParams, this.activeCompanyUniqueName).subscribe((res) => {
```
[the remark this answers is not in the document]
Comment (ravinderwalkover) [reply apps/web-giddh/src/app/mobile-home/mobile-home.component.ts:179]: yes

**Pull request merged**